### PR TITLE
Update to not modify req.url for getServerSideProps requests

### PR DESCRIPTION
**Pull request by ijjk**
This fixes an inconsistency where we would modify `req.url` during internal routing for `/_next/data` routes but the `req.url` wouldn't be modified when deploying in `serverless` mode. To fix the inconsistency this makes sure we don't modify the `req.url` with `next start` to match us not modifying it after being deployed.

There is a potential inconsistency with `getStaticProps` and `req.url` still as we keep the HTML and JSON in sync so depending on which endpoint is visited first on a revalidate would dictate which version is used. Should this be normalized for `getStaticProps` or is it better to not modify `req.url` in this case either? 🤔 

This also adds tests to ensure the `query` is correctly passed to `_app.getInitialProps` for dynamic pages and `getServerSideProps` along with ensuring the correct `req.url` is present.

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| buildDuration | 11.1s | 10.9s | -227ms |
| nodeModulesSize | 47.7 MB | 47.7 MB | ⚠️ +284 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5272.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.3 kB | 56.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.78 kB | 4.78 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.75 kB | 6.75 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| index.html gzip | 917 B | 917 B | ✓ |
| link.html gzip | 926 B | 926 B | ✓ |
| withRouter.html gzip | 915 B | 915 B | ✓ |
| Overall change | 2.76 kB | 2.76 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| buildDuration | 12.3s | 12.1s | -201ms |
| nodeModulesSize | 47.7 MB | 47.7 MB | ⚠️ +284 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5272.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.3 kB | 56.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.78 kB | 4.78 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.75 kB | 6.75 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _error.js gzip | 233 kB | 233 kB | ⚠️ +2 B |
| 404.html gzip | 1.32 kB | 1.32 kB | ✓ |
| hooks.html gzip | 956 B | 956 B | ✓ |
| index.js gzip | 232 kB | 233 kB | ⚠️ +277 B |
| link.js gzip | 243 kB | 243 kB | -60 B |
| routerDirect.js gzip | 241 kB | 241 kB | -70 B |
| withRouter.js gzip | 241 kB | 241 kB | ⚠️ +52 B |
| Overall change | 1.19 MB | 1.19 MB | ⚠️ +201 B |

</details>

</details>

File: test/integration/getserversideprops/test/index.test.js [hunk 1]
```diff
@@ -260,6 +260,46 @@ const runTests = (dev = false) => {
...
+
+    expect(appProps).toEqual({
+      url: curUrl,
+      query: { post: 'post-1' },
+      asPath: curUrl,
+      pathname: '/blog/[post]',
+    })
+  })
+
+  it('should have original req.url for /_next/data request', async () => {
```
Comment (timneutkens) [inline test/integration/getserversideprops/test/index.test.js:276]: This wouldn't work in serverless I believe 🤔

Comment (ijjk) [reply test/integration/getserversideprops/test/index.test.js:276]: Serverless mode is being tested here, which part doesn't seem like it would work?

Comment (timneutkens) [reply test/integration/getserversideprops/test/index.test.js:276]: I looked at it the wrong way 👍

**Review by timneutkens — APPROVED** [review]

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| buildDuration | 10.2s | 10.4s | ⚠️ +184ms |
| nodeModulesSize | 47.7 MB | 47.7 MB | ⚠️ +284 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..e3d8.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.3 kB | 56.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.78 kB | 4.78 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.78 kB | 6.78 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.5 kB | 51.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| index.html gzip | 918 B | 918 B | ✓ |
| link.html gzip | 927 B | 927 B | ✓ |
| withRouter.html gzip | 914 B | 914 B | ✓ |
| Overall change | 2.76 kB | 2.76 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| buildDuration | 11.5s | 11.3s | -161ms |
| nodeModulesSize | 47.7 MB | 47.7 MB | ⚠️ +284 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..e3d8.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.3 kB | 56.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.78 kB | 4.78 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.78 kB | 6.78 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.5 kB | 51.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _error.js gzip | 233 kB | 233 kB | -6 B |
| 404.html gzip | 1.32 kB | 1.32 kB | ✓ |
| hooks.html gzip | 957 B | 957 B | ✓ |
| index.js gzip | 233 kB | 233 kB | -4 B |
| link.js gzip | 243 kB | 243 kB | ⚠️ +25 B |
| routerDirect.js gzip | 241 kB | 241 kB | -137 B |
| withRouter.js gzip | 241 kB | 241 kB | ⚠️ +365 B |
| Overall change | 1.19 MB | 1.19 MB | ⚠️ +243 B |

</details>

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| buildDuration | 10.5s | 10.2s | -324ms |
| nodeModulesSize | 47.7 MB | 47.7 MB | ⚠️ +284 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..e3d8.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.3 kB | 56.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.78 kB | 4.78 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.78 kB | 6.78 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.5 kB | 51.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| index.html gzip | 918 B | 918 B | ✓ |
| link.html gzip | 927 B | 927 B | ✓ |
| withRouter.html gzip | 914 B | 914 B | ✓ |
| Overall change | 2.76 kB | 2.76 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| buildDuration | 11s | 10.8s | -216ms |
| nodeModulesSize | 47.7 MB | 47.7 MB | ⚠️ +284 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..e3d8.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.3 kB | 56.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.78 kB | 4.78 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.78 kB | 6.78 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.5 kB | 51.5 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary  | ijjk/next.js fix/gssp-req-url | Change |
| - | - | - | - |
| _error.js gzip | 233 kB | 233 kB | -228 B |
| 404.html gzip | 1.32 kB | 1.32 kB | ✓ |
| hooks.html gzip | 957 B | 957 B | ✓ |
| index.js gzip | 233 kB | 233 kB | -257 B |
| link.js gzip | 243 kB | 243 kB | ⚠️ +55 B |
| routerDirect.js gzip | 241 kB | 241 kB | -230 B |
| withRouter.js gzip | 241 kB | 241 kB | ⚠️ +3 B |
| Overall change | 1.19 MB | 1.19 MB | -657 B |

</details>

</details>

**Pull request merged**